package.json

```diff
@@ -34,6 +34,8 @@
     "@babel/preset-react": "^7.0.0",
     "@babel/register": "^7.4.4",
     "@webcomponents/url": "~0.7.1",
+    "acorn": "^6.0.0",
```

Comment:
Can you please add a comment why these were added?
Since it's not a direct dependency someone might try to remove them in the future.

Reply:
Comments aren't allowed in JSON so I think we need to rely on Git blame and additional CI checks here. I'm going to look at how we can catch this as part of the Fauxton CI rather than waiting until a CouchDB release to find the error.